Sources/SotoCore/HTTP/AsyncHTTPClient.swift

```diff
@@ -27,103 +27,41 @@ extension AsyncHTTPClient.HTTPClient {
     func execute(
         request: AWSHTTPRequest,
         timeout: TimeAmount,
-        on eventLoop: EventLoop,
         logger: Logger
-    ) -> EventLoopFuture<AWSHTTPResponse> {
-        let requestBody: AsyncHTTPClient.HTTPClient.Body?
-        var requestHeaders = request.headers
+    ) async throws -> AWSHTTPResponse {
+        let requestBody: HTTPClientRequest.Body?
 
         switch request.body.payload {
         case .byteBuffer(let byteBuffer):
-            requestBody = .byteBuffer(byteBuffer)
-        case .stream(let reader):
-            requestHeaders = reader.updateHeaders(headers: requestHeaders)
-            requestBody = .stream(length: reader.contentSize) { writer in
-                return writer.write(reader: reader, on: eventLoop)
-            }
-        case .empty:
+            requestBody = .bytes(byteBuffer)
+        /* case .asyncSequence(let sequence, let length):
+         requestBody = .stream(
+             sequence,
+             length: length.map { .known($0) } ?? .unknown
+         ) */
```

Comment:
Commented out code

Reply:
This is commented out while streaming requests don't work. It will be re-instated when I get them working